packages/primereact/package.json

```
@@ -0,0 +1,124 @@
+{
+  "name": "@rjsf/primereact",
+  "version": "6.0.0-alpha.0",
+  "description": "PrimeReact theme, fields and widgets for react-jsonschema-form",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "exports": {
```

This is needed for better ESM support
```suggestion
  "typings": "lib/index.d.ts",
  "type": "module",
  "exports": {
```